stable/gangway/README.md

```diff
@@ -35,6 +35,7 @@ At a minimum you *must* configure any of the values marked as **required** in th
 | Parameter                        | Description                                                                                                                                                                                                     | Default                                            |
 | -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- |
 | `affinity`                       | List of affinities (requires Kubernetes >=1.6)                                                                                                                                                                  | `{}`                                               |
+| `env`                   | Optional list of environment variables                                                                                                                                                                                           | `[]`                                               |
```

Comment:
Might want to add a note explaining that `GANGWAY_SESSION_SECURITY_KEY` is already defined and you cannot override.

Reply:
we could also add the `GANGWAY_SESSION_SECURITY_KEY` as default to the `values.yaml` so it stays consistent?

Reply:
I like the idea, but I think the problem will be this bit:

```
              name: {{ include "gangway.fullname" . }}-key
```

You can't include template fns. in `values.yaml`. I'd leave it and add a warning, nobody has ever asked for that to be made an option so I'm tempted to say lets cross that bridge when we come to it!

Reply:
yeah you are totally right - I will update the PR today

Reply:
srry, for the delay - this issue is still active for me and I'll try to make the change as soon as possible